### PR TITLE
update grails-core and gradle-grails-plugin to 6.2.2-SNAPSHOT

### LINKED ISSUE #377: Determine what adds the next snapshot to https://start.grails.org/
https://github.com/grails/grails-forge/actions/runs/11172670058 didn't seem to do it.

### PR DESCRIPTION
**Pull request by jamesfredley**
this PR is attempting to get 6.2.2-SNAPSHOT to show on https://start.grails.org/ as the 2nd option.

### CODE REVIEW

File: grails-forge-core/src/main/resources/pom.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
         <dependency>
             <groupId>org.grails</groupId>
             <artifactId>grails-gradle-plugin</artifactId>
-            <version>6.2.1</version>
+            <version>6.2.2-SNAPSHOT</version>
```
Comment (matrei) [inline grails-forge-core/src/main/resources/pom.xml]: Looking at the history, I don't think this should be updated to SNAPSHOT.

File: grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy [hunk 2]
```diff
@@ -43,8 +43,8 @@ class GradleSpec extends ApplicationContextSpec implements CommandOutputFixture
         settingsGradle.contains("mavenLocal()")
         settingsGradle.contains("maven { url \"https://repo.grails.org/grails/core/\" }")
         settingsGradle.contains("gradlePluginPortal()")
-        settingsGradle.contains("id \"org.grails.grails-web\" version \"6.2.1\"")
-        settingsGradle.contains("id \"org.grails.grails-gsp\" version \"6.2.1\"")
+        settingsGradle.contains("id \"org.grails.grails-web\" version \"6.2.2-SNAPSHOT\"")
+        settingsGradle.contains("id \"org.grails.grails-gsp\" version \"6.2.2-SNAPSHOT\"")
```
Comment (matrei) [inline grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy]: Looking at the history, I don't think this should be updated.

File: grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy [hunk 3]
```diff
@@ -76,10 +76,10 @@ class GradleSpec extends ApplicationContextSpec implements CommandOutputFixture
         settingsGradle.contains("mavenLocal()")
         settingsGradle.contains("maven { url \"https://repo.grails.org/grails/core/\" }")
         settingsGradle.contains("gradlePluginPortal()")
-        settingsGradle.contains("id \"org.grails.grails-web\" version \"6.2.1\"")
+        settingsGradle.contains("id \"org.grails.grails-web\" version \"6.2.2-SNAPSHOT\"")
```
Comment (matrei) [inline grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy]: Don't update this.

File: grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy [hunk 4]
```diff
@@ -76,10 +76,10 @@ class GradleSpec extends ApplicationContextSpec implements CommandOutputFixture
...
         settingsGradle.contains("id \"org.grails.plugins.views-markup\" version \"3.2.3\"")
         !settingsGradle.contains("id \"org.grails.plugins.views-json\" version \"3.2.3\"")
-        !settingsGradle.contains("id \"org.grails.grails-gsp\" version \"6.2.1\"")
+        !settingsGradle.contains("id \"org.grails.grails-gsp\" version \"6.2.2-SNAPSHOT\"")
```
Comment (matrei) [inline grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy]: Don't update this.

File: grails-forge-core/src/test/groovy/org/grails/forge/feature/grails/GrailsGradlePluginSpec.groovy [hunk 5]
```diff
@@ -16,8 +16,8 @@ class GrailsGradlePluginSpec extends BeanContextSpec implements CommandOutputFix
         final String gradleProps = output["gradle.properties"]
 
         then:
-        gradleProps.contains("grailsGradlePluginVersion=6.2.1")
-        gradleProps.contains("grailsVersion=6.2.1")
+        gradleProps.contains("grailsGradlePluginVersion=6.2.2-SNAPSHOT")
+        gradleProps.contains("grailsVersion=6.2.2-SNAPSHOT")
```
Comment (matrei) [inline grails-forge-core/src/test/groovy/org/grails/forge/feature/grails/GrailsGradlePluginSpec.groovy:20]: Only update the `grailsVersion`

File: grails-forge-core/src/test/groovy/org/grails/forge/feature/grails/GrailsGradlePluginSpec.groovy [hunk 6]
```diff
@@ -26,7 +26,7 @@ class GrailsGradlePluginSpec extends BeanContextSpec implements CommandOutputFix
                 .renderBuildSrc()
 
         then:
-        template.contains('implementation("org.grails:grails-gradle-plugin:6.2.1")')
+        template.contains('implementation("org.grails:grails-gradle-plugin:6.2.2-SNAPSHOT")')
```
Comment (matrei) [inline grails-forge-core/src/test/groovy/org/grails/forge/feature/grails/GrailsGradlePluginSpec.groovy]: Don't update this.

File: grails-forge-core/src/test/groovy/org/grails/forge/feature/grails/GrailsGradlePluginSpec.groovy [hunk 7]
```diff
@@ -36,7 +36,7 @@ class GrailsGradlePluginSpec extends BeanContextSpec implements CommandOutputFix
 
         expect:
         buildSrcBuildGradle != null
-        buildSrcBuildGradle.contains("implementation(\"org.grails:grails-gradle-plugin:6.2.1\")")
+        buildSrcBuildGradle.contains("implementation(\"org.grails:grails-gradle-plugin:6.2.2-SNAPSHOT\")")
```
Comment (matrei) [inline grails-forge-core/src/test/groovy/org/grails/forge/feature/grails/GrailsGradlePluginSpec.groovy]: Don't update this.

File: grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy [hunk 8]
```diff
@@ -59,9 +59,9 @@ class GradleSpec extends ApplicationContextSpec implements CommandOutputFixture
         settingsGradle.contains("mavenLocal()")
         settingsGradle.contains("maven { url \"https://repo.grails.org/grails/core/\" }")
         settingsGradle.contains("gradlePluginPortal()")
-        settingsGradle.contains("id \"org.grails.grails-web\" version \"6.2.1\"")
+        settingsGradle.contains("id \"org.grails.grails-web\" version \"6.2.2-SNAPSHOT\"")
```
Comment (matrei) [inline grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy]: Don't update this.

File: grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy [hunk 9]
```diff
@@ -59,9 +59,9 @@ class GradleSpec extends ApplicationContextSpec implements CommandOutputFixture
...
         settingsGradle.contains("id \"org.grails.plugins.views-json\" version \"3.2.3\"")
-        !settingsGradle.contains("id \"org.grails.grails-gsp\" version \"6.2.1\"")
+        !settingsGradle.contains("id \"org.grails.grails-gsp\" version \"6.2.2-SNAPSHOT\"")
```
Comment (matrei) [inline grails-forge-core/src/test/groovy/org/grails/forge/build/gradle/GradleSpec.groovy]: Don't update this.

**Review by matrei — CHANGES_REQUESTED** [review]
Looking at the history, this is my best guess. Only the `grailsVersion` in `gradle.properties` and `GrailsGradlePluginSpec` should be set to the next `SNAPSHOT` version.

File: grails-forge-core/src/main/resources/pom.xml (hunk 1 above)
Comment (jamesfredley) [reply grails-forge-core/src/main/resources/pom.xml]: I think this is going to fail the tests given our experience yesterday during the launch, but I am pushing it up now to test.  I think the difference is that grails-core:6.2.2-SNAPSHOT has been already been updated to point to grails-gradle-plugin:6.2.2-SNAPSHOT.

Comment (matrei) [reply grails-forge-core/src/main/resources/pom.xml]: Hmm, it seems the tests passed. Lets push it and see if we get `6.2.2-SNAPSHOT` as an option in the UI.

**Review by matrei — APPROVED** [review]

File: grails-forge-core/src/main/resources/pom.xml (hunk 1 above)
Comment (jamesfredley) [reply grails-forge-core/src/main/resources/pom.xml]: It passed the tests, so merging it with just the grailsVersion change.

**Comment by jamesfredley** [comment]
https://github.com/grails/grails-forge/actions/runs/11181613124

**Comment by jamesfredley** [comment]
That worked, https://start.grails.org/ has 6.2.2-SNAPSHOT

**Pull request merged**